### PR TITLE
Fix the Conditional logic for ansible Playbook function

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
The current logic doesn't capture the Error when ansible isn't available in the `$PATH` from where the playbook/kubetest2 command is run.
The following error goes unnoticed
```
exec: "ansible-playbook": executable file not found in `$PATH`
```
The `Playbook` func from pkg/ansible/ansible.go is returning a nil error and hence the kubetest2 deployer is proceeding further looking for kubeconfig. Thus the below **wrong error message**:
```
Error: failed to setKubeconfig: failed to locate the kubeconfig file: stat /root/kubetest2-plugins/ragni-wed2-vm/kubeconfig: no such file or directory
```
as mentioned by @RagniGarg97 in below comment.
https://github.ibm.com/powercloud/container-dev/issues/1452#issuecomment-29386246

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/32#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by RagniGarg97 — APPROVED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/32#pullrequestreview-633430354" title="Approved">RagniGarg97</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/32#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/ansible/ansible.go
Comment (mkumatag) [inline pkg/ansible/ansible.go:39]: You can directly mention the non zero here.. no need to declare a variable for that..

File: pkg/ansible/ansible.go
Comment (mkumatag) [inline pkg/ansible/ansible.go:37]: Not sure golang will work fine without any return statement in the end

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: pkg/ansible/ansible.go
Comment (Rajalakshmi-Girish) [reply pkg/ansible/ansible.go:37]: It did work. May be because all the internal conditions have a return.

**Comment by Rajalakshmi-Girish** [comment]
@mkumatag Can you please take a look?

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/32#pullrequestreview-666411958" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/32#pullrequestreview-633430354" title="Approved">RagniGarg97</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/32#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
